packages/inference/test/HfInference.spec.ts

```diff
@@ -835,7 +880,7 @@ describe.concurrent("HfInference", () => {
 				expect(res).toBeInstanceOf(Blob);
 			});
 
-			it("textToSpeech OuteTTS", async () => {
+			it.skip("textToSpeech OuteTTS", async () => {
```

Comment:
this one did not work in your local tests? When https://replicate.com/jbilcke/oute-tts is Warm it was working for us.

Maybe we can replace it with / add https://replicate.com/jaaari/kokoro-82m, it should be Warmer and faster, WDYT?

Reply:
Yeah it didn't work in my local tests so I skipped it. Looks like the model is cold.

I tried swapping it out with kokoro model but got this error:

```
   × HfInference > Replicate > textToSpeech hexgrad/Kokoro-82M 363ms
     → "- input: text is required\n"
```

Reply:
yep bc the schema i used is different from the one in `hexgrad/Kokoro-82M`. Given that one is more popular/Warmer, let's switch schemas, WDYT?